test/e2e/create_test.go

```diff
@@ -133,6 +133,53 @@ func (tc *testContext) testBYOHConfiguration(t *testing.T) {
 		t.Skip("BYOH testing disabled")
 	}
 
+	err := tc.configureBYOHWithMachineSets()
+	require.NoError(t, err, "error configuring BYOH with MachineSets")
+
+	t.Run("VM is configured by ConfigMap controller", func(t *testing.T) {
+		err := tc.waitForWindowsNodes(gc.numberOfBYOHNodes, false, false, true)
+		assert.NoError(t, err, "Windows node creation failed")
+	})
+}
+
+// configureBYOHWithMachineSets configures BYOH instances using MachineSets
```

Comment:
This function will prevent Machine CSRs from being approved in the future, right?
Lets make sure thats captured in the function comment.

Reply:
Makes sense.

Reply:
Addressed in https://github.com/openshift/windows-machine-config-operator/commit/26df7d86c978d8a4d324f497341af63d9fd25d3d